app/src/androidTest/java/org/mozilla/focus/activity/robots/BrowserRobot.kt

```diff
@@ -321,13 +323,18 @@ class BrowserRobot {
         var currentTries = 0
         while (currentTries++ < 3) {
```

Comment:
I think it looks ok. But would you mind refactoring this while loop in the same style we used lately with "for (i in 0..RETRY_COUNT)"?

Reply:
Sure, no problem !

Reply:
Done !